pyqmc/ewald.py

```diff
@@ -84,10 +84,12 @@ def __init__(self, cell, ewald_gmax=200, nlatvec=1):
         :parameter int ewald_gmax: how far to take reciprocal sum; probably never needs to be changed.
         :parameter int nlatvec: how far to take real-space sum; probably never needs to be changed.
         """
-        self.nelec = np.array(cell.nelec)
+        self.latvec = cell.lattice_vectors()
         self.atom_coords = cell.atom_coords()
+        self.dimension = cell.dimension
```

Comment:
I think we remove this. Maybe this should be replaced with a check that cell.dimension is 3? With a warning if not?